biojava-core/src/test/java/org/biojava/nbio/core/util/PrettyXMLWriterTest.java

```diff
@@ -68,7 +68,8 @@ void specialCharsAreEscaped() throws IOException {
         xw.print("<code>some literal xml </code>");
         xw.closeTag("dt");
 
-        assertEquals(EXPECTED, sw.toString());
+        if(! StringManipulationHelper.equalsToIgnoreEndline(EXPECTED, sw.toString()))
```

Comment:
Using asserts is more idiomatic (but no more correct). e.g. assertTrue(expected, actual, messageOnFail), here and elsewhere in the file
```
 assertTrue(StringManipulationHelper.equalsToIgnoreEndline(EXPECTED, sw.toString()),
        String.format("Strings are not equal (ignoring endline differences. expected: [%s], but it was:[%s]",
         EXPECTED, sw.toString()));
```

Reply:
OK. 👍